tsconfig.json

```diff
@@ -15,6 +15,7 @@
         "webpack.d.ts"
     ],
     "exclude": [
-        "node_modules"
+        "node_modules",
+        "decidim-comments/app/frontend/entry_test.ts"
```

Comment:
Are you sure about this?

Reply:
This was done by @beagleknight actually. Maybe he has more insights on why it was done?

Reply:
It's fine. The `entry_test.ts` should not belong to the typescript project because it will cause compiling issues.

Reply:
But why should it belong to it, if it has the `.ts` extension? 😕

Reply:
It has the `.ts` extension because is a typescript file but it doens't mean it must belong to the project. In other words that file is a support file for the testing suite not an actual test.